test/unit/handlers/lib/kafka/producer.test.js

```diff
@@ -258,7 +258,7 @@ Test('Producer', producerTest => {
         test.end()
       } catch (e) {
         test.ok(e instanceof Error)
-        test.ok(e.toString() === `Error: The following Producers could not be disconnected: [{"topic":"${topicNameFailure}","error":"No producer found for topic ${topicNameFailure}"}]`)
+        test.equal(e.message, `The following Producers could not be disconnected: [{"topic":"${topicNameFailure}","error":"No producer found for topic ${topicNameFailure}"}]`)
```

Comment:
Producer.getProducer() throws the following:

throw ErrorHandler.Factory.createInternalServerFSPIOPError('')

so instead of adding:

test.assert(error instanceof Errors.ValidationError)

added:

test.assert(e instanceof FSPIOPError)